src/components/configure/configuration/ConfigurationDialog.tsx

```diff
@@ -241,6 +266,13 @@ export default class ConfigurationDialog extends React.Component<
       >
         <DialogTitle id="draggable-dialog-title" style={{ cursor: 'move' }}>
           Configuration
+          <Button
+            onClick={() => {
+              this.onClickDownloadCheatSheet();
+            }}
+          >
+            download
+          </Button>
```

Comment:
@adamrocker It seems that you forgot this download button for testing. Could you remove this?

![Screenshot from 2021-03-02 06-50-45](https://user-images.githubusercontent.com/261787/109564256-297c8e80-7b24-11eb-8144-34e3cc218ad1.png)

Reply:
Sorry, this is a test button....　I forgot to remove. Let me clean up these test codes.

Reply:
@yoichiro I cleaned up all test codes in the dialog. Could you please confirm them again?